src/V3File.cpp

```diff
@@ -447,8 +446,7 @@ class VInFilterImp {
                      || errno == EWOULDBLOCK
 #endif
                 ) {
-                // cppcheck-suppress obsoleteFunctionsusleep
-                checkFilter(false); usleep(1000); continue;
+                checkFilter(false); V3Os::usleep(1000); continue;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This C routine is considered obsolete (as opposed to the shell command by the same name).   The interaction of this function with SIGALRM and other timer functions such as sleep(), alarm(), setitimer(), and nanosleep() is unspecified (CWE-676).  Use nanosleep(2) or setitimer(2) instead. ](https://app.codacy.com/app/wsnyder/verilator/pullRequest?prid=4719042)